cluster/operations/ldap.yml

```diff
@@ -11,7 +11,8 @@
     # Skip certificate verification. Default: false
     insecure_skip_verify: ((ldap_insecure_skip_verify))
     # The CA certificate for the LDAP auth provider’s endpoints.
-    ca_cert: ((ldap_ca_cert))
+    ca_cert: 
+      certificate: ((ldap_ca_cert))
```

Comment:
i don't think this is actually proper - it seems like the current ops file expects `ldap_ca_cert` to be a `certificate` typed variable

@brightzheng100 is that right?

Reply:
@brightzheng100 @vito Then you cannot read the certificate from a file, rather you would have to set the variable in a file and interpolate it. Both work but I believe reading the certificate from a file is a better solution.

Reply:
Yes, I agree with you @sandyg1.
Based on what I've tested [here](https://github.com/concourse/concourse-bosh-deployment/pull/139#issuecomment-466905299), it's better to make it more specific, even either way works.